tests/main/test_origin/test_ascend_like.py

```diff
@@ -11,10 +11,10 @@
 
 # Expected energy and latency for each workload defined above
 ens_lats = {
-    "zigzag/inputs/examples/workload/alexnet.onnx": (5738192980.375, 8728331),
-    "zigzag/inputs/examples/workload/mobilenetv2.onnx": (1913797698.5250015, 7426499),
-    "zigzag/inputs/examples/workload/resnet18.onnx": (1858697886.165, 3720129),
-    "zigzag.inputs.examples.workload.resnet18": (2408671233.7250004, 4804196),
+    "zigzag/inputs/examples/workload/alexnet.onnx": (5737868753.12, 8696023),
+    "zigzag/inputs/examples/workload/mobilenetv2.onnx": (1913561726.0000005, 7359650),
+    "zigzag/inputs/examples/workload/resnet18.onnx": (1860918012.2400002, 3698589),
+    "zigzag.inputs.examples.workload.resnet18": (2345967030.96, 4779555),
 }
 
```

Comment:
why do all these numbers change?

Reply:
In the old configuration, when a mapping is provided for a certain operator (e.g. C : 32), but the loop dimension is not present in the layer (so no C loop dimension), the unrolling would be reduced to 1.
In the new configuration, when such an invalid mapping is provided, it gets deleted (with a warning to the user), and replaced with a generated one. Thus, all tests results are slightly better. You can also visually see this: there are no spatial stalls with the new version.

Reply:
Aha, interesting

Reply:
@RobinGeens can you update the numbers for all working tests?

For the sake of progress, we can remove the other tests of files (they will be updated by @JiacongSun later).

Reply:
The numbers have been updated for all `test_origin ` tests. The others rely on Jiacong's code